.circleci/config.yml

```diff
@@ -453,7 +456,10 @@ workflows:
           filters:
             branches:
               ignore: /pull\/[0-9]+/
-      - integration-test-integration-airflow-2:
+      - integration-test-integration-airflow:
+          matrix:
+            parameters:
+              airflow-image: ['apache/airflow:2.1.3', 'apache/airflow:2.2.3', 'gcr.io/speedy-vim-308516/apache/airflow:latest']
```

Comment:
Nice 💯

Reply:
Does `gcr.io/speedy-vim-308516/apache/airflow` test Airflow on GCP?

Reply:
No, it just uses Google Container Registry as a source of Airflow 2.3's docker image.

Reply:
Shouldn't the tag be `2.3.x` instead of `latest` then?